### PR TITLE
AYS-420 | Refactor EmergencyEvacuationApplicationsList Response Body 

### PR DESCRIPTION
**Pull request by cagla88**
## Pull Request

**Description:**
EmergencyEvacuationApplicationsList response body edited.

**Changes Made:**
`applicantFirstName`, `applicantLastName`, `applicantPhoneNumber` information has been added to the fields returned in response.
New test case added for existing not `isInPerson` application

**Additional Information:**
Add any additional information that might be relevant to reviewers.

**Reviewer Instructions:**
Provide instructions or guidelines for reviewers to follow when assessing your pull request.

### Checklist

Before submitting your pull request, ensure the following:

- [x] **Title and Branch Naming Conventions:**
    - The pull request title follows the
      standard: [Pull Request Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be-api-automation/blob/main/CONTRIBUTING.md#pull-request-naming-conventions)
      .
    - The branch name follows one of the conventions outlined in
      the [Branch Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be-api-automation/blob/main/CONTRIBUTING.md#branch-naming-conventions)
      .

- [x] **Local Testing:**
    - I have added positive and negative test scenarios (if applicable)
    - All tests are passing.
    - I tagged all test as smoke or regression
    - I added newly added tests to related xml file in test suite

- [x] **Code Quality:**
    - The code is formatted according to the project's coding guidelines and style.
    - The code has been reviewed to ensure its quality.
    - The code does not contain any issues flagged by SonarLint.

- [x] **Documentation:**
    - Necessary documentation has been added or existing documentation has been updated.

- [x] **Reviewers and Assignees:**
    - Default reviewers have been assigned to this pull request.
    - Assignees have been added if necessary.

- [x] **Labels and Associations:**
    - No specific actions are required in the Labels and Associations section for this pull request.

- [x] **Reviewer Instructions:**
    - Provide instructions or guidelines for reviewers to follow when assessing your pull request.

### CODE REVIEW

**Review by htczrrsz — CHANGES_REQUESTED** [review]
Selam Çağla, 203. line ile 267. line'da ayni icerik var uyarısı alıyorum SonarLint'ten, sadece isimleri farklı gibi duruyor, bir de 251 ile 219. Sen de bir kontrol edebilir misin ?

File: src/test/java/org/ays/emergencyapplication/tests/EmergencyEvacuationApplicationsListTest.java [hunk 1]
```diff
@@ -52,7 +58,7 @@ public void testListingEmergencyEvacuationApplications() {
     }
 
     @Test(groups = {"Smoke", "Regression"})
-    public void testFilteringEvacuationApplicationsWithAnExistingApplicationData() {
+    public void testFilteringEvacuationApplicationsWithAnExistingIsInPersonApplicationData() {
```
Comment (Nbhtozcn) [inline src/test/java/org/ays/emergencyapplication/tests/EmergencyEvacuationApplicationsListTest.java:61]: method ismin "testFilteringEvacuationApplicationsByNonInPersonApplication()" seklinde olması daha anlamlı olur diye düşünüyorum.

**Comment by cagla88** [comment]
@htczrrsz selam, 203 te invalidCity testi varken 267 de invalidTargetCity testi var birbirlerinden farklı testler. Bende sonarLint te bir uyarı görünmüyor. Ek olarak bu pr ile belirttiğin testlerde bir düzenleme yapılmadı

File: src/test/java/org/ays/emergencyapplication/tests/EmergencyEvacuationApplicationsListTest.java (hunk 1 above)
Comment (cagla88) [reply src/test/java/org/ays/emergencyapplication/tests/EmergencyEvacuationApplicationsListTest.java]: @Nbhtozcn selam method ismini güncelledim tekrar review yapabilir misiniz?

File: src/test/java/org/ays/emergencyapplication/tests/EmergencyEvacuationApplicationsListTest.java [hunk 2]
```diff
@@ -92,6 +102,52 @@ public void testFilteringEvacuationApplicationsWithAnExistingApplicationData() {
...
+                .body("response.totalElementCount", equalTo(1))
+                .body("response.orderedBy[0].property", equalTo("createdAt"))
+                .body("response.orderedBy[0].direction", equalTo("DESC"))
+                .body("response.filteredBy.referenceNumber", equalTo(EmergencyEvacuationApplicationDataSource.findLastCreatedReferenceNumber()))
+                .body("response.filteredBy.sourceCity", equalTo(applicationPayload.getSourceCity()))
+                .body("response.filteredBy.sourceDistrict", equalTo(applicationPayload.getSourceDistrict()))
+                .body("response.filteredBy.seatingCount", equalTo(applicationPayload.getSeatingCount()))
+                .body("response.filteredBy.targetCity", equalTo(applicationPayload.getTargetCity()))
+                .body("response.filteredBy.targetDistrict", equalTo(applicationPayload.getTargetDistrict()))
+                .body("response.filteredBy.statuses[0]", equalTo("PENDING"))
```
Comment (agitrubard) [inline src/test/java/org/ays/emergencyapplication/tests/EmergencyEvacuationApplicationsListTest.java:147]: Eğer varsa `EmergencyEvacuationApplicationStatus` sınıfını bu kısımlarda `"PENDING"` yerine kullanabiliriz. Eğer yoksa oluşturup yazabiliriz çünkü buradaki isimlendirmeler değiştirilmek istendiğinde ekstra efor yaratabilir

**Review by agitrubard — APPROVED** [review]

**Review by htczrrsz — APPROVED** [review]

**Pull request merged**